src/System.IO.Packaging/tests/Tests.cs

```diff
@@ -3644,28 +3644,35 @@ public void SetEmptyPropertyToNull()
         public void CreateWithFileAccessWrite()
         {
             string[] fileNames = new [] { "file1.txt", "file2.txt", "file3.txt" };
+            const string RelationshipType = "http://schemas.microsoft.com/relationships/contains";
+            const string PartRelationshipType = "http://schemas.microsoft.com/relationships/self";
 
-            using (Stream stream = new MemoryStream())
+            using (Stream stream = new FileStream("write.zip", FileMode.Create))
```

Comment:
you want to change this to use a temporary location to make it also work under UAP.

Reply:
Doh. Let me undo. This was some lazy hacking to persist outpu.